x-pack/plugin/ql/src/test/resources/mapping-multi-field-variation.json

```diff
@@ -27,6 +27,14 @@
                         }
                     }
                 },
+                "text": {
```

Comment:
Why do you need this and not use the `string` field right above?

Reply:
Because the `string` field has multiple exact keywords. If we use it, the SQL result will be an error
```
{
  "error": {
    "root_cause": [
      {
        "type": "sql_illegal_argument_exception",
        "reason": "Multiple exact keyword candidates available for [string]; specify which one to use"
      }
    ],
    "type": "sql_illegal_argument_exception",
    "reason": "Multiple exact keyword candidates available for [string]; specify which one to use"
  },
  "status": 500
}
```
Searching for exact values only works with text field with a single keyword.